app/models/concerns/aliquotable.rb

```diff
@@ -34,5 +34,23 @@ module Aliquotable
     has_many :derived_aliquots, -> { where(aliquot_type: :derived) },
              as: :source, class_name: 'Aliquot',
              dependent: :nullify, inverse_of: :source
+
+    # Method to calculate the used volume
+    def used_volume
+      derived_aliquots.sum(:volume)
```

Comment:
Im not sure how this will behave it derived_aliquots is empty. This may need to be `derived_aliquots.sum(&:volume)` See 'Safe Navigation Operator section' https://ruby-doc.org/core-2.6/doc/syntax/calling_methods_rdoc.html.

Reply:
Thank you. Learned a new Ruby thing 👍